Scripts/script.js

```diff
@@ -1,56 +1,57 @@
 (function(module) {
-
+  var names = {};
+  names.all = [];
     //project Constructor function
   function Project (project) {
     Object.keys(project).forEach(function(property, index, keys) {
       this[property] = project[property];
     },this);
   }
-  Project.projectList = [];
   //code to create a method to check localStorage for json object and prepend it to the dom
   Project.fetchAll = function() {
+    Project.projectList = [];
     var dataList;
     if (localStorage.storageProjects) {
       dataList = JSON.parse(localStorage.storageProjects);
+      dataList[0].projectTitle = names.all[0];
+      dataList[1].projectTitle = names.all[1];
```

Comment:
Best would be to dynamically load all of the data by iterating over the list if it exists. Hardcoding it like this makes things more difficult in the future when you add a new project.